package.json

```diff
@@ -7,19 +7,17 @@
     "data": "npm run data-airlines && npm run data-aircrafts",
     "data-airlines": "mkdir -p data && curl -o data/airlines.csv https://raw.githubusercontent.com/jpatokal/openflights/master/data/airlines.dat",
     "data-aircrafts": "mkdir -p data && curl -o data/aircrafts.zip https://junzis.com/adb/download && unzip data/aircrafts.zip aircraft_db.csv -d data && mv data/aircraft_db.csv data/aircrafts.csv && rm -rf data/aircrafts.zip",
-    "postinstall": "npx electron-rebuild",
+    "postinstall": "./node_modules/.bin/MagicMirror-rebuild",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Thomas Bouron <tbouron@gmail.com>",
   "license": "Apache 2.0",
   "dependencies": {
     "csv-parse": "^4.6.5",
+    "magicmirror-rebuild": "^1.0.5",
```

Comment:
I think this is a regular dependency rather than a dev-only one. My understanding is that we need to run `MagicMirror-rebuild` after all installations in order for this to work properly?

Reply:
Ah I have a slightly different understanding: `dependencies` are for the ones required at runtime versus `devDependencies` are for compile time. But it's a nitpick really, it doesn't change anything in the end so fine with that @mattdy ;)